### PR TITLE
0.9.14 + node update + arm by default

### PR DESCRIPTION
**Pull request by colemickens**
* add 0.9.14 (latest)
* update to node5 (stable)
* pull the node version to an env variable
* switch to arm by default

### CODE REVIEW

File: 0.9.14/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    rm -rf /var/lib/apt/lists/* && \
+    curl https://deb.nodesource.com/${NODEJS_APT_ROOT}/pool/main/n/nodejs/nodejs_${NODEJS_VERSION}-1nodesource1~jessie1_amd64.deb > node.deb && \
+      dpkg -i node.deb && \
+      rm node.deb && \
+      npm install --global azure-cli@${AZURE_CLI_VERSION} && \
+      azure --completion >> ~/azure.completion.sh && \
+      echo 'source ~/azure.completion.sh' >> ~/.bashrc && \
+      azure
+
+RUN bash -c "mkdir -p ~/.azure; echo '{"mode":"arm"}' > ~/.azure/config.json"
```
Comment (ahmetalpbalkan) [inline 0.9.14/Dockerfile]: Hmm why don't you just run `azure config mode arm` in the line above?

Comment (colemickens) [inline 0.9.14/Dockerfile]: You're right. I had changed this while investigating something else.

(I am mounting ~/.azure into the container which means that I override whatever mode I set in here anyway. I still think it should default to `arm`.)

**Comment by colemickens** [comment]
Updated with `azure config more arm` instead of writing the `config.json` manually.

**Comment by ahmetalpbalkan** [comment]
LGTM. @devigned are you going to carry this PR or should I just merge/release?

**Comment by devigned** [comment]
@ahmetalpbalkan I'm going to run through this with @yugangw-msft today.

**Comment by colemickens** [comment]
And can someone with access (@ahmetalpbalkan?) to the Microsoft account on Docker Hub tag and push this new version?

Thanks.

**Comment by ahmetalpbalkan** [comment]
@yugangw-msft is on it.

**Comment by yugangw-msft** [comment]
@ahmetalpbalkan , @colemickens, my account doesn't have the permission to queue a new build, i have let @amarzavery do it this time.

**Comment by ahmetalpbalkan** [comment]
@yugangw-msft ah I'm sorry I added you to the wrong team. the link I sent over email should be accessible now.

**Comment by yugangw-msft** [comment]
@ahmetalpbalkan @colemickens , one extra comment is on the node version. For azure-cli installers, we have picked the node 4.2.4, and we would like be consistent and use the same version for docker image as well. 
The reason with 4.2.4 is simply for the reliability.It is the one with longest LTS tag, being more reliable than node 5.x with less feature works. It will enter maintenance mode in next Apr, and we will then switch to next the LTS thereafter. [This post](https://medium.com/@nodesource/essential-steps-long-term-support-for-node-js-8ecf7514dbd#.tu9cchqwj) has more explanations.

**Comment by colemickens** [comment]
I had wondered if there would be push back on that. I'm fine with bumping it down to 4.2.4. I can send a PR for it...

**Pull request merged**